astropy/io/votable/tests/test_converter.py

```diff
@@ -16,7 +16,6 @@
 def test_invalid_arraysize():
     with pytest.raises(exceptions.E13):
         field = tree.Field(None, name="broken", datatype="char", arraysize="foo")
-        converters.get_converter(field)
```

Comment:
Sub-package maintainer should verify that these deletions are valid.

Reply:
@tboch @tomdonaldson @pllim Looks like you are the sub-package maintainers according to the astropy team page but never got notified in the initial PR. Can you verify these deletions are valid?

Reply:
I had a closer look at this file, so I can now say that the deleted lines in this file were never executed to begin with, so deleting them is in fact the correct thing to do. What I don't know is whether or not the tests in question should be checking the state of `converters`. Currently they do not.

Reply:
> What I don't know is whether or not the tests in question should be checking the state of `converters`.

I am not sure either. But if it is never executed, then maybe okay to remove. I would wait a few more days in case @tomdonaldson objects. Thanks for your patience!

Or if you cannot wait, defer `io.votable` as a follow-up PR.

Reply:
@eerovaher and @pllim I agree that it's OK to remove these two `converters.get_converter()` lines.  As mentioned, they were not being executed.  I was unable to find another code path within `converters.get_converter()` that would be worth testing for the exception, so this may have been superfluous from the beginning.  It's also possible that some previous behavior had the checks being more lazy and not triggered until `converters.get_converter()`.  I wasn't able to see that easily, but even if true it would be fine to remove these lines.